src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java

```diff
@@ -556,6 +556,20 @@ public Object invokeMethod(String name, Object args) {
         assertEvaluate(new GenericWhitelist(), 3, "'foo bar baz'.tokenize('ba').size()");
     }
 
+    @Issue("JENKINS-33023")
+    @Test public void enums() throws Exception {
+        String script = "enum Thing {\n"
+            + "  FIRST(\"The first thing\");\n"
+            + "  String description;\n"
+            + "  public Thing(String description) {\n"
+            + "    this.description = description;\n"
+            + "  }\n"
+            + "}\n"
+            + "Thing.values()[0].description\n";
+        String expected = "The first thing";
+        assertEvaluate(new BlanketWhitelist(), expected, script);
```

Comment:
Also I would expect `.values()` to be automatically whitelisted; the current test does not indicate whether it is or not.